xcodecommand/fallbackRunner.go

```diff
@@ -0,0 +1,38 @@
+package xcodecommand
+
+import (
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/log"
+	version "github.com/hashicorp/go-version"
+)
+
+type FallbackRunner struct {
+	runner         Runner
+	fallbackRunner Runner
+	logger         log.Logger
+}
+
+func NewFallbackRunner(runner Runner, logger log.Logger, commandFactory command.Factory) *FallbackRunner {
+	return &FallbackRunner{
+		runner:         runner,
+		fallbackRunner: NewRawCommandRunner(logger, commandFactory),
+		logger:         logger,
+	}
+}
+
+func (sel *FallbackRunner) CheckInstall() (*version.Version, error) {
```

Comment:
nit: what is `sel`?

Reply:
It was a leftover, short for selector, but naming is hard.